hack/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 Pressinfra SRL.
+Copyright 2019 Pressinfra SRL.
```

Comment:
Is it necessary?

Reply:
Why not?

Reply:
https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year